### PR TITLE
Updates to CZML for model node animations

### PR DESCRIPTION
**Pull request by gbeatty**

### CODE REVIEW

File: Source/DataSources/ModelVisualizer.js [hunk 1]
```diff
@@ -235,9 +269,7 @@ define([
     }
 
     function onModelReady(model) {
-        model.activeAnimations.addAll({
```
Comment (gbeatty) [inline Source/DataSources/ModelVisualizer.js]: This removes turning on all model animations when the model is loaded. Still need to find a way to manually turn on animations when loading models through the entity framework so the Sandcastle models example works as it did before

Comment (gbeatty) [inline Source/DataSources/ModelVisualizer.js]: Commit 9359a01 fixes this issue by providing a runAnimations property on ModelGraphics

Comment (mramato) [inline Source/DataSources/ModelVisualizer.js]: Thanks.

**Comment by shunter** [comment]
related czml-writer PRs.  AnalyticalGraphicsInc/czml-writer#101 and AnalyticalGraphicsInc/czml-writer#102

I made additional changes on top of Greg's changes and so I will be opening a new PR to reflect those changes in Cesium as well that will supersede this PR.  FYI.

**Comment by shunter** [comment]
replaced by #3316

**Pull request closed without merging**